writlarge/settings_shared.py

```diff
@@ -1,60 +1,75 @@
+# flake8: noqa
 # Django settings for writlarge project.
 import os.path
+import platform
+import sys
 from ccnmtlsettings.shared import common
 
+
 project = 'writlarge'
 base = os.path.dirname(__file__)
 
 locals().update(common(project=project, base=base))
 
+
+if platform.linux_distribution()[1] == '16.04':
+    # 15.04 and later need this set, but it breaks
+    # on trusty.
+    # yeah, it's not really going to work on non-Ubuntu
+    # systems either, but I don't know a good way to
+    # check for the specific issue. Anyone not running
+    # ubuntu will just need to set this to the
+    # appropriate value in their local_settings.py
```

Comment:
If Ubuntu 14.04 is the only place you've seen this breaking, would it be better to reverse this check? Like, set `SPATIALITE_LIBRARY_PATH` **unless** this is running on Ubuntu 14.04? Does this work on your mac os x setup?